legion/sdk/legion/sdk/clients/edi_aggregated.py

```diff
@@ -196,14 +225,14 @@ def parse_resources_file_with_one_item(path: str) -> LegionCloudResourceUpdatePa
     return resources.changes[0]
 
 
-def apply(updates: LegionCloudResourcesUpdateList, edi_client: RemoteEdiClient, is_removal: bool) -> ApplyResult:
+async def apply(updates: LegionCloudResourcesUpdateList, async_edi_client: AsyncRemoteEdiClient, is_removal: bool) -> ApplyResult:
```

Comment:
Did you try to run the robot tests? 
We use this function in the CLI package. It doesn't expect to get a coroutine.